kuryr_kubernetes/controller/managers/prometheus_exporter.py

```diff
@@ -130,6 +155,19 @@ def _create_metrics(self):
             ' on Subnet', labelnames={'subnet_id', 'subnet_name'},
             registry=self.registry)
 
+        self.lbs_members_count = prometheus_client.Gauge(
+            'kuryr_critical_lb_members_count', 'Amount of members per '
+            'critical Load Balancer pool',
+            labelnames={'lb_name', 'lb_pool_name'},
+            registry=self.registry)
+
+        self.lbs_state = prometheus_client.Enum(
+            'kuryr_critical_lb_state', 'Critical Load Balancer State',
+            labelnames={'lb_name'},
+            states=['ERROR', 'ACTIVE', 'DELETED', 'PENDING_CREATE',
+                    'PENDING_UPDATE', 'PENDING_DELETE'],
```

Comment:
I'm not sure, if it's intentional, but for every loadbalancer with certain label (in this case it would be 'namespace/lbname') this will create 6 metrics for Enum type. Those metrics will stay forever (or at least during the service lifetime), which in busy environment could easily grow to hundreds of them.

Reply:
It's intentional, we're monitoring only 2 critical load balancers. So, we'll have 2* ['ERROR', 'ACTIVE', 'DELETED', 'PENDING_CREATE', 'PENDING_UPDATE', 'PENDING_DELETE']